include/superior_mysqlpp/utils.hpp

```diff
@@ -75,4 +75,224 @@ namespace SuperiorMySqlpp
     {
         return streamify(streamify(stream, arg), std::forward<Args>(args)...);
     }
+
+    namespace detail
+    {
+        /**
+         * Function information flags
+         */
+        enum : uint64_t
+        {
+            FunctionFlag_IsConst = 1ULL << 0ULL,
+            FunctionFlag_IsVolatile = 1ULL << 1ULL,
+            FunctionFlag_IsLvalueRefCtx = 1ULL << 3ULL,
```

Comment:
Are you sure this is the correct term? I don't know myself, but in any case, I feel these flags (or appropriate FunctionInfo "fields") deserve comments.